src/decorators/JSONField.ts

```diff
@@ -0,0 +1,29 @@
+// tslint:disable-next-line: no-var-requires
+const GraphQLJSONObject = require('graphql-type-json').GraphQLJSONObject;
+import { Field } from 'type-graphql';
+import { Column } from 'typeorm';
+
+import { composeMethodDecorators, MethodDecoratorFactory } from '../utils';
+
+interface JSONFieldOptions {
+  nullable?: boolean;
+}
+
+export function JSONField(args: JSONFieldOptions = {}): any {
+  const nullableOption = args.nullable === true ? { nullable: true } : {};
+
+  // These are the 2 required decorators to get type-graphql and typeorm working
+  const factories = [
+    // We explicitly say string here because when we're metaprogramming without
```

Comment:
Where does this say string? Not sure what the comment means

Reply:
removed comment